tools/TestFx/Coverage/AnalyzeTestCoverage.ps1

```diff
@@ -191,11 +191,13 @@ foreach ($moduleName in $allModules) {
         $cvgBaseline | Export-Csv -Path $cvgBaselineCsv -Encoding utf8 -NoTypeInformation -Append -Force
     }
 
-    Write-Host "##[section]`"$moduleName`" total commands # : $totalCommandsCount" -ForegroundColor Green
-    Write-Host "##[section]`"$moduleName`" tested commands # : $totalTestedCommandsCount" -ForegroundColor Green
-    Write-Host "##[section]`"$moduleName`" test coverage % : $cvgCommand" -ForegroundColor Green
+    if ($CalcBaseline.IsPresent) {
```

Comment:
should this "if ($CalcBaseline.IsPresent)" combine with the previous conditional statement?

Reply:
This is another display issue for test coverage. It is very simple, so updated together in this PR.